Clicker/SectionControllers/SpaceSectionController.swift

```diff
@@ -21,6 +27,12 @@ class SpaceSectionController: ListSectionController {
     }
     
     override func cellForItem(at index: Int) -> UICollectionViewCell {
+        if noResponses {
+            let cell = collectionContext?.dequeueReusableCell(of: NoResponsesCell.self, for: self, at: index) as! NoResponsesCell
+            cell.setNeedsUpdateConstraints()
+            cell.configure(with: spaceModel.space)
```

Comment:
You should call `configure` before calling `setNeedsUpdateConstraints`. The reason we put constraints in `updateConstraints` is if we need to pass in some data to the cell that will be used in making our constraints.